try/run_central_instances.py

```diff
@@ -0,0 +1,2341 @@
+    """
+    if not input_list:
+        return ""
+    if not isinstance(input_list[0], six.string_types):
+        input_list = [str(item) for item in input_list]
+
+    if max_width:
+        mystr = ", ".join(input_list)
+        return fold_string(mystr, max_width)
+
+    return "\n".join(input_list)
+
+
+def fold_string(input_string, max_width):
+    """
+    Fold a string within a maximum width.
+
+      Parameters:
+
+        input_string:
+          The string of data to go into the cell
+        max_width:
+          Maximum width of cell.  Data is folded into multiple lines to
+          fit into this width.
+
+      Return:
+          String representing the folded string
+    """
+    new_string = input_string
+    if isinstance(input_string, six.string_types):
+        if max_width < len(input_string):
+            # use textwrap to fold the string
+            new_string = textwrap.fill(input_string, max_width)
+
+    return new_string
+
+
+def get_profile_name(org_vm, profile_inst):
+    """
+    Get the org, name, and version from the profile instance and
+    return them as a tuple.
+    Returns: tuple of org, name, vers
+
+    Raises:
+      TypeError: if invalid property type
+      ValueError: If property value outside range
+    """
+    try:
+        org = org_vm.tovalues(profile_inst['RegisteredOrganization'])
+        name = profile_inst['RegisteredName']
+        vers = profile_inst['RegisteredVersion']
+        return org, name, vers
+
+    except TypeError as te:
+        print('ORG_VM.TOVALUES FAILED. inst=%s, Exception %s' %
+              (profile_inst, te))
+    except ValueError as ve:
+        print('ORG_VM.TOVALUES FAILED. inst=%s, Exception %s' %
+              (profile_inst, ve))
+    return 'ERR', 'ERR', 'ERR'
+
+
+def profile_name(org_vm, profile_inst, short=False):
+    """
+    Return Org, Profile, Version as a string from the properties in the
+    profile instance.  The returned form is the form Org:Name:Version or
+    Org:Name if the optional argument short is TRUE
+    """
+    try:
+        name_tuple = get_profile_name(org_vm, profile_inst)
+    except Exception as ex:  # pylint: disable=broad-except
+        print('GET_FULL_PROFILE_NAME exception %sm tuple=%r inst=%s' %
+              (ex, name_tuple, profile_inst))
+        return("UNKNOWN")
+    if short:
+        return "%s:%s" % (name_tuple[0], name_tuple[1])
+
+    return "%s:%s:%s" % (name_tuple[0], name_tuple[1], name_tuple[2])
+
+
+def path_wo_ns(obj):
+    """
+    Return path of an instance or instance path without host or namespace.
+    Creates copy of the object so the original is not changed.
+    """
+    if isinstance(obj, pywbem.CIMInstance):
+        path = obj.path.copy()
+    elif isinstance(obj, pywbem.CIMInstanceName):
+        path = obj.copy()
+    else:
+        assert False
+
+    path.host = None
+    path.namespace = None
+    return path
+
+
+class ServerDefinitionFileError(Exception):
```

Comment:
I don't think the `ServerDefinitionFileError` class should be copied, but instead imported.

Because the script is invoked as a script, its directory does not count as a package even if it contains an `__init__.py` file. So you have to specify the path starting from the main directory:
```
form tests.servers.server_file import ServerDefinitionFileError
```

Reply:
Dropped completely since we use the ServerDefinitionFile from the tests now